jsk_pcl_ros_utils/scripts/xyz_to_screenpoint.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+
+import rospy
+
+from sensor_msgs.msg import CameraInfo
+from geometry_msgs.msg import PointStamped
+
+from image_geometry import PinholeCameraModel
+
+class XYZToScreenPoint(object):
+    def __init__(self):
+        self.cameramodels = PinholeCameraModel()
+        self.is_camera_arrived = False
+        self.header = None
+
+        # publish info
+        self.pub = rospy.Publisher("~output", PointStamped, queue_size=1)
+
+        rospy.Subscriber('~input/camera_info', CameraInfo, self.camera_info_cb)
+        rospy.Subscriber('~input', PointStamped, self.point_stamped_cb)
+
+    def camera_info_cb(self, msg):
+        self.cameramodels.fromCameraInfo(msg)
+        self.is_camera_arrived = True
+        self.header = msg.header
+
+    def point_stamped_cb(self, msg):
+        if not self.is_camera_arrived:
+            return
+        point = (msg.point.x, msg.point.y, msg.point.z)
+        u, v = self.cameramodels.project3dToPixel(point)
+        rospy.logdebug("u, v : {}, {}".format(u, v))
+        pub_msg = PointStamped()
+        pub_msg.header = self.header
```

Comment:
The header.frame_id should be self.header.frame_id, but header.stamp should be input msg's header.stamp.
It can be helpful for message synchronization.